ansible/roles/spdashboard/templates/parameters.yml.j2

```diff
@@ -10,3 +10,15 @@ parameters:
     mailer_user: null
     mailer_password: null
     secret: {{ spdashboard_symfony_secret }}
+    metadata_url_timeout: 30
+    session_max_absolute_lifetime: 3600
+    session_max_relative_lifetime: 600
+    logout_redirect_url: https://www.surf.nl/over-surf/werkmaatschappijen/surfnet
+    administrator_team: {{ spdashboard_administrator_team }}
+    saml_sp_publickey: "{{ spdashboard_saml_sp_publickey }}"
+    saml_sp_privatekey: "{{ spdashboard_saml_sp_privatekey }}" 
+    saml_metadata_publickey: "{{ spdashboard_saml_metadata_publickey }}"
+    saml_metadata_privatekey: "{{ spdashboard_saml_metadata_privatekey }}" 
+    saml_remote_idp_entity_id: "{{ spdashboard_saml_remote_idp_entity_id }}"
+    saml_remote_idp_sso_url: "{{ spdashboard_saml_remote_idp_sso_url }}"
+    saml_remote_idp_certificate: "{{ spdashboard_saml_remote_idp_certificate }}"
```

Comment:
I might be overlooking something here, but I think the `spdashboard_saml_remote_idp_certificate` Ansible variable does not exist.

Reply:
This variable is in the environment repository. It's left out here, since deploying a development VM does not deploy the application itself.